index.js

```diff
@@ -162,6 +210,18 @@ const line = []
 
 // ***** Scope & Closures - Question 2 *****
 // *** Uncomment the lines below to test
+
+function ticketNumberGeneratorFunc() {
+  let current_num = 1
+  let current_line = []
+
+  function takeATicketNumber(current_line) {
+    current_line.push(current_num)
+    console.log(`Welcome. You are ticket number ${current_line.length}`)
+    current_num++
+  }
+  return takeATicketNumber
+}
```

Comment:
- If `takeATicketNumber` accepts an array, does `current_line` on line 216 serve a purpose? Try taking 216 out & testing it out
- Better variable names for `current_num` and `current_line` are `currentNum` and `currentLine`